api/src/main/java/marquez/MarquezApp.java

```diff
@@ -5,8 +5,13 @@
 
 package marquez;
 
+import co.elastic.clients.elasticsearch.ElasticsearchClient;
```

Comment:
ElasticSearch derived from open-source project which currently forked into OpenSearch to remain community driven. Is it possible to bring some abstraction layer here to allow other implementations and vendors? For AWS users, https://aws.amazon.com/opensearch-service/ may seem a more natural case to use.

Reply:
I will look into making this swap out.